routers/router.go

```diff
@@ -0,0 +1,11 @@
+package routers
+
+import (
+	"goMarketer/controllers"
+
+	"github.com/astaxie/beego"
```

Comment:
Please refer to this Slack message: https://nimble-co.slack.com/archives/C018FE1GPDG/p1607935898014600. We should be using https://github.com/beego/beego/ as it's the original repository for Beego. Sorry for the confusion on the IC requirements.

Reply:
This was done by Bee CLI as default @olivierobert 😀

Reply:
I just press `bee new` and it generates that, let me change it to `https://github.com/beego/beego/`

Reply:
I don't know what wrong with that but we can't go with `github.com/beego/beego` @olivierobert, I tried with that and having the  `Not Found Module` error

![image](https://user-images.githubusercontent.com/11751745/102182374-bd07d100-3ede-11eb-96b2-12889d8aa70b.png)

The core Beego team also giving the example to using `github.com/astaxie/beego` at https://github.com/beego/beego/issues/4308#issuecomment-726116425